test/val/val_id_test.cpp

```diff
@@ -177,9 +185,62 @@ std::string BranchConditionalTail = R"(
                OpFunctionEnd
 )";
 
+std::string ValidateIdWithMessage::make_message(const char* msg) {
+  const bool use_friendly_names = GetParam();
+  if (use_friendly_names) {
+    return msg;
+  }
+
+  std::string message(msg);
+  std::string result;
+
+  size_t next = 0;
+  while (next < message.size()) {
+    // Parse 'num[%name]'
+    size_t open_quote = message.find('\'', next);
+    fprintf(stderr, "%zu\n", open_quote);
```

Comment:
This method shouldn't print to stderr,  I don't think.
That's noisy.

Reply:
Oops, left-over debug code. Removed.